### PR TITLE
Release notes for 2024.2 release

### PR DESCRIPTION
**Pull request by HenKlei**
Please take a look @pymor/main which features should be included where and write the release notes for those things you feel responsible for.

Rendered notes are [here](https://docs.pymor.org/release-notes-2024-2/release_notes/all.html#pymor-2024-2-december-20-2024).

### CODE REVIEW

**Comment by HenKlei** [comment]
Ping @pymor/all

**Comment by sdrave** [comment]
@artpelling, @maxbindhak, any comments? (Release is planned for tomorrow ..)

File: docs/source/release_notes/2024.2.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+provided.
+
+
+Backward incompatible changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Dropped Python 3.8 support
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Python version 3.8 was dropped `[#2358] <https://github.com/pymor/pymor/pull/2358>`_.
+The latest tested version is now Python 3.9.
```
Comment (artpelling) [inline docs/source/release_notes/2024.2.rst:52]:
```suggestion
The oldest tested version is now Python 3.9.
```

Comment (sdrave) [reply docs/source/release_notes/2024.2.rst:52]: Thank! Actually, I'd prefer

```suggestion
The oldest supported version is now Python 3.9.
```

**Pull request merged**